gir-files/GstGL-1.0.gir

```diff
@@ -2558,8 +2558,70 @@ display's object lock held.</doc>
         </array>
       </field>
     </record>
-    
-    
+    <class name="GLDisplayEGL" c:symbol-prefix="gl_display_egl" c:type="GstGLDisplayEGL" parent="GLDisplay" glib:type-name="GstGLDisplayEGL" glib:get-type="gst_gl_display_egl_get_type" glib:type-struct="GLDisplayEGLClass">
```

Comment:
I wonder how we can at some point get a .gir file that has EGL but also Windows, macOS, iOS and all the other things... probably need some fixup script to merge all these somehow

Reply:
Can .gir files be auto-generated in Windows, MacOS, iOS, etc.?

The other option would be craft the xml and inserted with ``fix.sh`` and "force"  ``gir``  to auto-generate the bindings.

Though, I guess we have always the option to add those bindings manually in the gstreamer-rs/gstreamer-gl crate.

Reply:
They can, yes. Let's worry about that when someone needs it :)